src/multiselect/__tests__/multiselect.test.tsx

```diff
@@ -259,6 +259,103 @@ test('does not render token group when no tokens are present', () => {
   expect(wrapper.findByClassName(tokenGroupStyles.root)).toBeNull();
 });
 
+describe('Token group', () => {
```

Comment:
Added additional unit tests for the token group along with the ones for covering the new aria-label properties.